R/class-union.R

```diff
@@ -19,7 +19,7 @@ setClass(
 
 # CrmPackClass-classunion
 
-#CohortSize
+# CohortSize
 setClassUnion(
   "CrmPackCohortSize1",
   c(
```

Comment:
could we automatically find those child classes instead of hardcoding them? Just thinking how we can simplify maintenance

Reply:
We could construct a tree like hierarchy, I suppose.  It could be done.  What we have now is an MVP.  But I do like the idea of automatically updating as the package evolves.  

Originally, I thought we could simply list the first ancestor in each tree: `setClassUnion("CrmPackClass", c("CohortSize", "Data", "Design", "Increments, ...)`.  But that didn't work.

Let me investigate...